sim/core/character.go

```diff
@@ -466,6 +466,13 @@ func (character *Character) Finalize() {
 		})
 	}
 
+	if character.Class == proto.Class_ClassHunter {
+		character.Env.RegisterPostFinalizeEffect(func() {
+			character.focusBar.setupFocusThresholds()
+		})
+	}
```

Comment:
Does this have to be here and not in the class file? I see the rogue one was also here but we can probably move it out as well? I'm not a fan of class specific code in the core

Reply:
I guess it doesn't. What do you think about my solution in the latest commit?